src/commands/pack/macos.ts

```diff
@@ -89,6 +89,16 @@ fi
 
 #remove application source distribution
 [ -e "/usr/local/lib/${config.dirname}" ] && rm -rf "/usr/local/lib/${config.dirname}"
+
+#remove application data directory
+[ -e "${config.dataDir}" ] && rm -rf "${config.dataDir}"
+
+#remove application cache directory
+[ -e "${config.cacheDir}" ] && rm -rf "${config.cacheDir}"
+
+#remove application config directory
+[ -e "${config.configDir}" ] && rm -rf "${config.configDir}"
```

Comment:
Doesn't this correspond to the `.sfdx`/`.sf` folders? I thought we didn't want to remove those on uninstall.

Reply:
@RodEsp No, this is a different folder: `~/.config/sf[dx]`

The config folder used by sfdx-core is `~/.sf[dx]`

Reply:
Ah, got'cha. Getting confused 😅 ...